source/utilities.tex

```diff
@@ -14759,6 +14826,150 @@
+is used to both interpret \placeholder{S} and construct \placeholder{E}.
+
+\begin{itemize}
+\item
+\unicode{0022}{quotation mark} (\tcode{"}) is appended to \placeholder{E}.
+
+\item
+For each code unit sequence \placeholder{X} in \placeholder{S} that either
+encodes a single character,
+is a shift sequence, or
+is a sequence of ill-formed code units,
+processing is in order as follows:
+
+\begin{itemize}
+\item
+If \placeholder{X} encodes a single character \placeholder{C}, then:
+
+\begin{itemize}
+\item
+If \placeholder{C} is one of the characters in \tref{format.escape.sequences},
+then the two characters shown as the corresponding escape sequence
+are appended to \placeholder{E}.
+
+\item
+Otherwise, if \placeholder{C} is not \unicode{0020}{space} and
+
+\begin{itemize}
+\item
+\placeholder{CE} is a Unicode encoding and
+\placeholder{C} corresponds to either
+a UCS scalar value whose Unicode property \tcode{General_Category}
+has a value in the groups \tcode{Separator (Z)} or \tcode{Other (C)} or to
+a UCS scalar value which has the Unicode property \tcode{Grapheme_Extend=Yes},
+as described by table 12 of UAX \#44, or
+
+\item
+\placeholder{CE} is not a Unicode encoding and
+\placeholder{C} is one of an implementation-defined set
+of separator or non-printable characters
+\end{itemize}
+
+then the sequence \tcode{$\backslash$u\{\placeholder{hex-digit-sequence}\}}
+is appended to \placeholder{E},
+where \placeholder{hex-digit-sequence}
+is the shortest hexadecimal representation
+of \placeholder{C} using lower-case hexadecimal digits.
+
+\item
+Otherwise, \placeholder{C} is appended to \placeholder{E}.
+\end{itemize}
+
+\item
+Otherwise, if \placeholder{X} is a shift sequence,
+the effect on \placeholder{E} and further decoding of \placeholder{S}
+is unspecified.
+It is recommended that
+a shift sequence be represented in \placeholder{E}
+such that the original code unit sequence of \placeholder{S}
+can be reconstructed.
+
+\item
+Otherwise (\placeholder{X} is a sequence of ill-formed code units),
+each code unit \placeholder{U} is appended to \placeholder{E} in order
+as the sequence \tcode{$\backslash$x\{\placeholder{hex-digit-sequence}\}},
+where \placeholder{hex-digit-sequence}
+is the shortest hexadecimal representation of \placeholder{U}
+using lower-case hexadecimal digits.
+\end{itemize}
+
+\item
+Finally, \unicode{0022}{quotation mark} (\tcode{"})
+is appended to \placeholder{E}.
+\end{itemize}
+%
+\begin{floattable}{Mapping of characters to escape sequences}{format.escape.sequences}{ll}
+\topline
+\lhdr{Character} & \rhdr{Escape sequence} \\ \rowsep
+\unicode{0009}{character tabulation} &
+\tcode{$\backslash$t}
+\\ \rowsep
+%
+\unicode{000a}{line feed} &
+\tcode{$\backslash$n}
+\\ \rowsep
+%
+\unicode{000d}{carriage return} &
+\tcode{$\backslash$r}
+\\ \rowsep
+%
+\unicode{0022}{quotation mark} &
+\tcode{$\backslash$"}
+\\ \rowsep
+%
+\unicode{005c}{reverse solidus} &
+\tcode{$\backslash\backslash$}
```

Comment:
This should be `\textbackslash` and not in math font:

```suggestion
\tcode{\textbackslash t}
\\ \rowsep
%
\unicode{000a}{line feed} &
\tcode{\textbackslash n}
\\ \rowsep
%
\unicode{000d}{carriage return} &
\tcode{\textbackslash r}
\\ \rowsep
%
\unicode{0022}{quotation mark} &
\tcode{\textbackslash "}
\\ \rowsep
%
\unicode{005c}{reverse solidus} &
\tcode{\textbackslash\textbackslash}
```

Reply:
Which looks like this:

![Uploading image.png…]()

Reply:
fixed